Lab1_Operationalizing_Pytorch_with_Mlflow/src/bert-classifier/requirements.txt

```
@@ -1,18 +1,19 @@
-torch==1.7.1+cpu
+# both conda and requirements
 mlflow>=1.12.0
-torchserve
-torch-model-archiver
-numpy
-torchserve
-torch-model-archiver
-numpy
-scikit-learn
-cloudpickle==1.6.0
-boto3
+sklearn
 transformers
 torchtext
 torch
```

Maybe we should keep the CPU version since we have no GPUs available.
```suggestion
torch==1.7.1+cpu
```

---

Will be resolved by base image using serve install_dependencies